server.js

```diff
@@ -1,55 +1,53 @@
-const express = require('express');
+const express = require("express");
 
-const messages = require('./messages');
+const messages = require("./messages");
 
 const server = express();
 
-const staticHandler = express.static('public');
+const staticHandler = express.static("public");
 
 server.use(staticHandler);
 
-const bodyParser = express.urlencoded({extended:true});
+const bodyParser = express.urlencoded({ extended: true });
 
 let uid = Object.keys(messages).pop(); //5?
 
-server.post('/', bodyParser, (req,res) => {
+server.post("/", bodyParser, (req, res) => {
+  // Two alternative methods for getting a unique ID:
 
-    // Two alternative methods for getting a unique ID:
+  // const lastKey = Object.keys(messages).pop();
+  //messages[lastKey+1] = req.body;
 
-    // const lastKey = Object.keys(messages).pop();
-    //messages[lastKey+1] = req.body;
+  uid++;
+  //console.log(uid);
+  //console.log(typeof uid);
+  messages[uid] = req.body;
 
-    uid++;
-    //console.log(uid);
-    //console.log(typeof uid);
-    messages[uid] = req.body;
-
-    res.redirect('/');
-})
-
-server.get('/', (req,res)=>{
-
-    let myHtml = '';
+  res.redirect("/");
+});
 
-    for (const [key, value] of Object.entries(messages)) {
-        //console.log(key, value);
-        let userName = value.name;
-        let userId = key;
-        let userText = value.text;
-        //console.log(userId, userName, userText)
-        myHtml += `
-        <div>
-            <h4>${userName}</h4>
-            <p>${userText}</p> 
-            <form action="/delete" method="POST">
+server.get("/", (req, res) => {
+  let myHtml = "";
+
+  for (const [key, value] of Object.entries(messages)) {
+    //console.log(key, value);
+    let userName = value.name;
+    let userId = key;
+    let userText = value.text;
+    //console.log(userId, userName, userText)
+    myHtml += `
+        <div id="message-display">
+            <h4 id="username">${userName}</h4>
+            <p id="posted-message">${userText}</p> 
+            <form action="/delete" method="POST" id="delete-message">
             <label>
-                <button name="messageToDelete" value="${userId}">Delete Me</button>
+                <button name="messageToDelete" value="${userId}" id="delete-button">Delete Me</button>
```

Comment:
Because these elements will appear on the page many times, we either need unique IDs, or to use classes instead.

Minju and Peter have already started using classes in their testing branch, so if you can change all the ``id="...`` stuff to ``class="..."`` instead, that would be great :)